angular/projects/spark-angular/src/lib/components/sprk-tabs/sprk-tabs.component.ts

```diff
@@ -0,0 +1,362 @@
+import {
+  AfterContentInit,
+  Component,
+  ContentChildren,
+  ElementRef,
+  HostListener,
+  Input,
+  QueryList,
+  Renderer2,
+} from '@angular/core';
+import * as _ from 'lodash';
+import { SprkTabsPanelDirective } from '../../directives/sprk-tabs/sprk-tabs-panel/sprk-tabs-panel.directive';
+import { SprkTabsButtonDirective } from '../../directives/sprk-tabs/sprk-tabs-button/sprk-tabs-button.directive';
+
+@Component({
+  selector: 'sprk-tabs',
+  template: `
+    <div
+      [ngClass]="getClasses()"
+      [attr.data-id]="idString"
+      [attr.data-analytics]="analyticsString"
+    >
+      <div class="sprk-c-Tabs__buttons" role="tablist">
+        <ng-content select="[sprkTabsButton]"></ng-content>
+      </div>
+      <ng-content select="[sprkTabsPane]"></ng-content>
```

Comment:
sprkTabsPanel - missing the 'l'

Reply:
good catch! I also fixed this in `sprk-tabbed-navigation`